api-admin/src/main/java/bio/terra/pearl/api/admin/controller/enrollee/EnrolleeController.java

```diff
@@ -75,8 +75,7 @@ public ResponseEntity<Object> requestKit(
     } catch (PepperException e) {
       log.error("Error requesting sample kit from Pepper", e);
       // In the case of a PepperException, we can do better than this because we'll likely know what
-      // Pepper was unhappy
-      // about, such as an address failing to validate.
```

Comment:
Some auto-reformatter kept splitting the first comment line at 80 characters. Didn't notice this in my prior PR.